### PR TITLE
Sscs 9031 carers allowance camel case fix

### PR DESCRIPTION
**Pull request by jackmaloney**
https://tools.hmcts.net/jira/browse/SSCS-9031

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/sscs/ccd/domain/BenefitTest.java [hunk 1]
```diff
@@ -15,4 +19,43 @@ public void givenAValidBenefitString_thenReturnTrue() {
...
+
+    @Test
+    @Parameters({
+            "PIP, Personal Independence Payment (PIP)",
+            "ESA, Employment and Support Allowance (ESA)",
+            "UC, Universal Credit (UC)",
+            "JSA, Job Seekers Allowance (JSA)",
+            "DLA, Disability Living Allowance (DLA)"
+    })
+    public void givenABenefitCodeWithAcronym_thenBuildLongBenefitNameDescriptionWithAcronym(String benefitCode, String expected) {
```
Comment (danhamilton) [inline src/test/java/uk/gov/hmcts/reform/sscs/ccd/domain/BenefitTest.java:31]: Does this exercise the scenario where there is no Acronym? e.g. Carers Allowance?

Comment (jackmaloney) [reply src/test/java/uk/gov/hmcts/reform/sscs/ccd/domain/BenefitTest.java:31]: That's covered in the test below (givenABenefitCodeWithNoAcronym_thenBuildLongBenefitNameDescriptionWithNoAcronym) although thinking about it, maybe i don't need to split into separate tests. What you think?

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/Benefit.java [hunk 2]
```diff
@@ -77,4 +89,14 @@ public static Boolean isBenefitTypeValid(String field) {
         }
         return false;
     }
+
+    public static String getLongBenefitNameDescriptionWithOptionalAcronym(String code, boolean isEnglish) {
+        Benefit benefit = getBenefitByCode(code);
+
```
Comment (rasikasri) [inline src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/Benefit.java:95]: Good to handle, if benefit == null as it is possible value to be returned by getBenefitByCode() and we get all benefits types in some application

Comment (jackmaloney) [reply src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/Benefit.java:95]: Isn't that handled in the getBenefitByCode method above? It throws an exception...

Comment (rasikasri) [reply src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/Benefit.java:95]: Yes, true. I mixed up the methods.

**Review by rasikasri — APPROVED** [review]

**Pull request merged**